src/type_resolve_helpers.ts

```diff
@@ -740,13 +740,28 @@ export function createTypeLiteral(children: IPropDesc[], parent?: IPropDesc): ts
         const opt = node.prop.optional ? ts.createToken(ts.SyntaxKind.QuestionToken) : undefined;
         const t = node.children.length ? createTypeLiteral(node.children, node) : resolveType(node.prop.type);
 
-        members.push(ts.createPropertySignature(
+        const property = ts.createPropertySignature(
             undefined,      // modifiers
             node.name,      // name
             opt,            // questionToken
             t,              // type
             undefined       // initializer
-        ));
+        );
+
+        // !parent ensures we are dealing with a top-level typedef.
+        // So that the tsd-doc is added at the property level.
+        if(!parent && (node.prop.description || node.prop.defaultvalue)) {
```

Comment:
@englercj Is there a better way I should be checking this?  The other option is threading the `doclet` through the calls so that I have access to `doclet.kind` here, which should be trivial to do but didn't want to make a bigger change before checking with you.

Reply:
I think this check is reasonable